updater/lib/dependabot/updater/operations/create_security_update_pull_request.rb

```diff
@@ -197,6 +195,20 @@ def check_and_create_pull_request(dependency)
           # Report this error to the backend to create an update job error
           raise
         end
+
+        sig { params(existing_pr: T.nilable(PullRequest)).returns(String) }
```

Comment:
Why is this nilable and then on line we check that it must exist? Can we get rid of the nilable?